README.md

```diff
@@ -1313,6 +1321,7 @@ MSc. Thesis, Air Force Institute of Technology, USA, 2019. [PDF](https://apps.dt
 - [Western Sydney University’s International Centre for Neuromorphic Systems (ICNS)](https://www.westernsydney.edu.au/icns), Australia.
 - [Perception and Robotics Group](http://prg.cs.umd.edu/) at University of Maryland (UMD).
 - [Intel Labs](https://www.intel.com/content/www/us/en/research/overview.html), [Mike Davies](https://newsroom.intel.com/news/bobbleheads-tool-test-neuromorphic-mind/#gs.nndw9d) (Intel’s neuromorphic computing program leader).
+- [RTC-lab Sevilla](http://www.rtc.us.es/), University of Seville, SPAIN
```

Comment:
Perhaps you want to add "Real Time Communication Laboratory" to show what RTC means?